### PR TITLE
Ideas 2 add cell hover

### LINKED ISSUE #6: Create cell hover display count
When a cell in the heatmap is hovered, display a count of the number of posts.

### PR DESCRIPTION
**Pull request by justin0022**
Resolves #6.

### CODE REVIEW

File: _ideas/heatmap/intermediate/2-cell-hover-count.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Heatmap - Intermediate
+
+## Task
+When a cell in the heatmap is hovered, display a count of the number of posts.
+
+## Overview
+To display a count on hover, you'll need to use D3's  to the heatmap cells. You may find [this guide](https://medium.com/@kj_schmidt/show-data-on-mouse-over-with-d3-js-3bf598ff8fc2) helpful.
```
Comment (markoprodanovic) [inline _ideas/heatmap/intermediate/2-cell-hover-count.md]: Typo? 
> you'll need to use D3's  to the heatmap cells

Guide is good resource though

**Review by markoprodanovic — CHANGES_REQUESTED** [review]
Looks good overall! 
Typo on line 7 of **2-cell-hover-count.md**?

**Comment by justin0022** [comment]
> Typo on line 7 of **2-cell-hover-count.md**?

Good catch, fixed.

**Comment by justin0022** [comment]
@all-contributors please add @justin0022 for doc, ideas, code, eventOrganizing, review

File: _ideas/heatmap/intermediate/2-cell-hover-count.md (hunk 1 above)
Comment (alisonmyers) [reply _ideas/heatmap/intermediate/2-cell-hover-count.md]: Resource https://medium.com/@kj_schmidt/show-data-on-mouse-over-with-d3-js-3bf598ff8fc2 uses D3.js version 4.6.0. Is this a problem?

Comment (justin0022) [reply _ideas/heatmap/intermediate/2-cell-hover-count.md]: I think this is okay, because the API for the mouseover hasn't changed!

**Pull request merged**